app/views/content/home/_directory-variant.html.erb

```diff
@@ -0,0 +1,36 @@
+<section class="homepage-feature image-blocks">
```

Comment:
The three images in this directory section are missing alt text - would one of you be able to add this in @gazcarr @philspray @EllieNodder ?

Reply:
Images are still being flagged as missing alt text @philspray 

Alt text is defined in a config file (`/config/images.yml`) - so that we only have to write it once if the image is used on multiple pages.

I think you just need to add the alt text to this config file and then delete it from on page (but @ethax-ross can confirm)

Reply:
Not sure why this is flagging. I added the alt text to the config file. I've tweaked it to see if that fixes it.